apps/store/extensions/assets/default/pages/list.jag

```diff
@@ -153,7 +153,7 @@ require('/modules/store.js').exec(function(ctx) {
     var output = am.render(assets, page).list();
     output.paging = assetApi.assetsPaging(assets, type, ctx.request);
     var qtemp = ctx.request.getParameter('q') || '';
-    var qjson = parse('{'+qtemp+'}');
-    output.searchQuery = qjson.name||'';
+    var searchQuery = assetApi.formatSearchQuery(qtemp);
+    output.searchQuery = searchQuery||'';
```

Comment:
Thanks @mushthaq33 , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/1cfd4a532c0b6fcbb158ab5da98f004afac2612c)